### PR TITLE
Add polyfill for angle brackets <LinkTo>

### PR DESCRIPTION
**Pull request by simonihmig**
Partially implements #66. A follow-up PR for `<Input>` should be pretty straightforward...

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -6,18 +6,18 @@ module.exports = {
...
     'eslint:recommended',
     'plugin:ember/recommended',
-    'prettier',
+    // 'prettier',
   ],
   env: {
     browser: true
   },
   rules: {
-    'prettier/prettier': 'error',
```
Comment (simonihmig) [inline .eslintrc.js]: I haven't figured this out yet...

Without it the build is failing with prettier running on `/addon`:

```
  - broccoliBuilderErrorStack: Error: ENOENT: no such file or directory, scandir '/Users/simonihmig/Projects/ember-angle-bracket-invocation-polyfill/addon/addon/helpers'
    at Object.readdirSync (fs.js:790:3)
    at traverseFolder (/Users/simonihmig/Projects/ember-angle-bracket-invocation-polyfill/node_modules/prettier/index.js:32564:18)
    at findRoot (/Users/simonihmig/Projects/ember-angle-bracket-invocation-polyfill/node_modules/prettier/index.js:32580:10)
    at maybeParse (/Users/simonihmig/Projects/ember-angle-bracket-invocation-polyfill/node_modules/prettier/index.js:32592:16)
    at editorconfigSyncNoCache (/Users/simonihmig/Projects/ember-angle-bracket-invocation-polyfill/node_modules/prettier/index.js:32605:35)
    at editorconfigSyncNoCache (/Users/simonihmig/Projects/ember-angle-bracket-invocation-polyfill/node_modules/prettier/index.js:28538:20)
    at /Users/simonihmig/Projects/ember-angle-bracket-invocation-polyfill/node_modules/prettier/index.js:34796:14
    at Array.map (<anonymous>)
    at _resolveConfig (/Users/simonihmig/Projects/ember-angle-bracket-invocation-polyfill/node_modules/prettier/index.js:34795:40)
    at Function.resolveConfig.sync (/Users/simonihmig/Projects/ember-angle-bracket-invocation-polyfill/node_modules/prettier/index.js:34831:12)
  - codeFrame: ENOENT: no such file or directory, scandir '/Users/simonihmig/Projects/ember-angle-bracket-invocation-polyfill/addon/addon/helpers'
  - errorMessage: Build Canceled: Broccoli Builder ran into an error with `broccoli-persistent-filter:EslintValidationFilter` plugin. 💥
```

I already tried upgrading related dependencies, but without success so far. Any ideas? 🤔

**Comment by rwjblue** [comment]
Remove `ember-cli-eslint`, add `eslint`.

File: .eslintrc.js (hunk 1 above)
Comment (rwjblue) [reply .eslintrc.js]: Remove `ember-cli-eslint`, add `eslint`.

Comment (rwjblue) [reply .eslintrc.js]: https://github.com/ember-cli/ember-cli-eslint/issues/235 for background (and https://github.com/emberjs/rfcs/blob/master/text/0121-remove-ember-cli-eslint.md).

**Comment by simonihmig** [comment]
This now includes #73, which should be merged first, and I'll rebase this afterwards. Other than that this should be ready for review!

**Comment by rwjblue** [comment]
Ok, the other two PR’s are landed. I think CI should be good to go with a rebase...

File: lib/ast-link-to-transform.js [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+    let visitor = {
+      ElementNode(node) {
+        if (node.tag.toLowerCase() === 'linkto') {
+          let { children, blockParams, attributes } = node;
+          let params = [];
+          let helperParams = [];
+
+          let route = attributes.find(({ name }) => name === '@route');
+          let model = attributes.find(({ name }) => name === '@model');
+          let models = attributes.find(({ name }) => name === '@models');
```
Comment (simonihmig) [inline lib/ast-link-to-transform.js]: We should probably put an assertion here that either `model` or `models` is passed, but not both, right? (as specified in the RFC)

Anything else we should cover? (an empty `route` is valid?)

**Comment by simonihmig** [comment]
Rebased and green now! :tada:

File: lib/ast-link-to-transform.js (hunk 2 above)
Comment (rwjblue) [reply lib/ast-link-to-transform.js]: Ya, empty route is fine. Assertion for both model and models is good.

**Comment by rwjblue** [comment]
Thanks for working on this!! Traveling atm but will try to get a release for these out soon.

**Pull request merged**